projects/aca-shared/src/lib/services/app.service.ts

```diff
@@ -177,6 +179,13 @@ export class AppService implements OnDestroy {
     });
 
     this.overlayContainer.getContainerElement().setAttribute('role', 'region');
+
+    const isMobileSwitchEnabled: boolean = this.config.get<boolean>('mobileAppSwitch.enabled', false);
+    if (isMobileSwitchEnabled) {
+      this.acaMobileAppSwitcherService.checkForMobileApp();
+    } else {
+      sessionStorage.removeItem('mobile_notification_expires_in');
```

Comment:
your `AcaMobileAppSwitcherService` is responsible dealing with session storage, I suggest having a method "reset" in the service that calls this "removeItem", as main application should not be responsible for cleanup.

You can then just call "this.acaMobileAppSwitcherService.reset()";

Reply:
made method reset in AcaMobileAppSwitcherService and called in else part for cleanup